src/cloud/architecture/pro-architecture.md

```diff
@@ -119,7 +119,7 @@ The three gateways map to the three servers in your Production environment clust
 
 ### Backup and disaster recovery
 
-Your Pro plan backup and recovery approach uses a high-availability architecture combined with full-system backups. We replicate each Project—all data, code, and assets—across three separate AWS Availability Zones, each zone with a separate data center.
+Your Pro plan backup and recovery approach uses a high-availability architecture combined with full-system backups. We replicate each Project—all data, code, and assets—across three separate AWS or Azure Availability Zones, each zone with a separate data center.
```

Comment:
There is a Production Tech stack image on this page that include AWS zone. It's an older graphic, so I don't know if you would have time to create a new one. You could change the stem sentence for it if you think it is important.